docs/game_event.md

```diff
@@ -1,66 +1,167 @@
+| [description](#description)  | VARCHAR(255) | SIGNED     |     | YES  | NULL    |        | Description of the event displayed in console                                                                                        |
+| [world_event](#worldevent)   | TINYINT      | UNSIGNED   |     | NO   |         |        | 0 if normal event, 1 if world event                                                                                                  |
+| [announce](#announce)        | TINYINT      | UNSIGNED   |     | YES  | 2       |        | 0 dont announce, 1 announce, 2 value from config                                                                                     |
 
 **Field Descriptions**
 
 ### eventEntry
 
 Entry of the event. Keep it as low as possible and prevent making holes in the list. Higher the max id is, the more memory will be used to store the event data.
 
-### start\_time
+### start_time
 
 Absolute start date of the event. The event will start occurring only if the local time at the server is after the one set here.
 
-### end\_time
+### end_time
 
 Absolute end date of the event. The event will stop occurring if the local time at the server is after the one set here.
 
 ### occurrence
 
 Number of minutes between 2 occurrences of the event. (2880 = 2 days, 1440 = 1 day, etc)
 
+{% include warning.html content="Value can't be 0 or the server will crash." %}
+
 ### length
 
 Number of minutes the event will last after the start of the occurrence. (2880 = 2 days, 1440 = 1 day, etc)
 This value must be lower than occurrence one or the event will never stop.
 
+
 ### holiday
 
-Holiday ID from  [Holidays DBC file](Holidays). This is sent to the client to update the calender.
+Holiday ID from  [Holidays DBC file](holidays). This is sent to the client to update the calender.
 
 ### description
 
 String containing the name of the event displayed in console each time it starts or stops.
 
-### world\_event
+### world_event
 
-This is a boolean field that determines if this game event is a world event or not. 0 = normal event, 1 = world event. For the world event to work, you need to at a minimum, populate game\_event\_condition and game\_event\_quest\_condition.
+This is a boolean field that determines if this game event is a world event or not. 0 = normal event, 1 = world event. For the world event to work, you need to at a minimum, populate [game_event_condition](game_event_condition) and [game_event_quest_condition](game_event_quest_condition).
 
 ### announce
+
+| Value | Description                                        |
+| ----- | -------------------------------------------------- |
+| 0     | Don't announce the Event                           |
+| 1     | Announce the description of the Event to the world |
+| 2     | Use the `event.announce` settings from the config  |
+
+### Database Content
```

Comment:
delete this. The database isnt static and updating this everytime something changes is just bad.

Reply:
Done.